scheduler/task_queue_persister.go

```diff
@@ -33,6 +33,7 @@ func PersistTaskQueue(distro string, tasks []task.Task, distroQueueInfo model.Di
 			GroupMaxHosts:       t.TaskGroupMaxHosts,
 			GroupIndex:          t.TaskGroupOrder,
 			Version:             t.Version,
+			Author:              t.ActivatedBy,
```

Comment:
I kind of wonder if people will get confused by calling the task activator the "author" in this case because anyone (user or automated process) can activate the task in any patch, not just the patch author. I don't necessarily have a better name for this field, but I'm wondering if you have alternative name ideas so it's less ambiguous.